gradle-mvn-push.gradle

```diff
@@ -235,4 +236,19 @@ afterEvaluate { project ->
       archives javadocJar
     }
   }
+
+  publishing {
+    publications {
+      library(MavenPublication) {
+        groupId GROUP
+        artifactId POM_ARTIFACT_ID
+        version VERSION_NAME + "-SNAPSHOT"
```

Comment:
i made a PR recently where i removed this suffix from `gradle.properties` file inside test folder. Would you be able to verify that my earlier change is safe? I had an issue with testing because the `mavenLocal` was not recognizing the version with "SNAPSHOT" suffix, which is why I made that change.

Reply:
Hm, this logic is like "You can specify any version you want, but we always put `-SNAPSHOT` after it, if it's a local publication." I think that sounds ~right. Having this be user-configurable in `gradle.properties` sounds the most flexible, but also possible to present confusing outcomes like you mention @raphkim